addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/tsmodelgen/TypeScriptModelsGenerator.java

```diff
@@ -269,6 +270,31 @@ private ModelDescriptor createModelDescriptor(Class<? extends WindupVertexFrame>
                 modelRelation.methodsPresent.addAll(methodInfo.methodsPresent);
                 modelDescriptor.addRelation(modelRelation);
             }
+
+            setInProperties:
+            {
```

Comment:
I think it would be better to have these in separate methods instead of named blocks.